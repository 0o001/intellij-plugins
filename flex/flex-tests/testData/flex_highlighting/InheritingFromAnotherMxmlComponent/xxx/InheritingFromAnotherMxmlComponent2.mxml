<mx:Application xmlns:mx="http://www.adobe.com/2006/mxml">
  <mx:Script>
    public function foo():void {}
    public function superMethod():void {}
  </mx:Script>
  <mx:Button id="mySuperField"/>
</mx:Application>