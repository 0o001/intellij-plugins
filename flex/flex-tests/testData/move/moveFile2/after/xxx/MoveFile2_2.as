package xxx {
  class MoveFile2_2 {
      var x:MoveFile2;
  }
}