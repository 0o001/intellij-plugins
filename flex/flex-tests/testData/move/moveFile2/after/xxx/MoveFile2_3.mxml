<mx:Application xmlns:mx="http://www.adobe.com/2006/mxml">
  <mx:Script>
      var x:MoveFile2;
  </mx:Script>
</mx:Application>