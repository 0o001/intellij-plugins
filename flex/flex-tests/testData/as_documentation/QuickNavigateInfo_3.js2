package xxx {
  public class AAA {
    public static function yyy(p:Object):AAA {}
  }
}

import xxx.AAA
AAA.y<caret>yy(1)