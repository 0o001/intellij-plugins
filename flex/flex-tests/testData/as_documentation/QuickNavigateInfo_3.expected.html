public static function xxx.<a href="psi_element://AAA">AAA</a>.yyy(p:Object):AAA